src/contact.ts

```diff
@@ -55,9 +64,8 @@ export type ContactQueryFilter = {
 
 /**
  * Class Contact
- * blabla...
- * **IMPORTANT**
  *
+ * `Contact` is `Sayable`
```

Comment:
I do not agree. Document is not the same as Code.

Examples can help users to understand easier how to use it rather than the API reference.

Sometimes you think it's obviously but for others are not, that's because you are too familiar with Wechaty.

I think to add examples for all API functions is necessary, but I'll leave this decision to you.

Please add examples to the APIs which you think is needed, leave it without example as you like.